Tools/setup/arch.sh

```diff
@@ -119,6 +123,43 @@ if [[ $INSTALL_SIM == "true" ]]; then
 		ant \
 		jdk8-openjdk \
 		;
+
+	# Gazebo setup
+	if [[ $INSTALL_GAZEBO == "true" ]]; then
+		echo
+		echo "Installing gazebo and dependencies for PX4 simulation"
+
+		# PX4 gazebo simulation dependencies
+		sudo pacman -S --noconfirm --needed \
+			eigen3 \
+			hdf5 \
+			opencv \
+			protobuf \
+			vtk \
+			yay \
+			;
+
+		# enable multicore gazebo compilation
+		sudo sed -i '/MAKEFLAGS=/c\MAKEFLAGS="-j'$(($(grep -c processor /proc/cpuinfo)+2))'"' /etc/makepkg.conf
+
+		# install gazebo from AUR
+		yay -S gazebo --noconfirm
+
+		# fix incompatible compile flag to disable default testing that leads to build error
+		# see https://bitbucket.org/ignitionrobotics/ign-cmake/issues/62/compilation-failing-when-performing
+		pushd ~/.cache/yay/ignition-cmake/
+		sed -i 's/-DENABLE_TESTS_COMPILATION:BOOL=False/-DBUILD_TESTING=OFF/g' PKGBUILD
```

Comment:
Strange, I didn't have this problem.

Reply:
The package maintainer screwed up something in the meantime. I didn't understand the process of how to contribute but I'll send him a message and we can take it out as soon as he fixed it.

Reply:
Update: I commented on the AUR to hopefully get the ignition-cmake fix in such that we can skip the workaround in the future: https://aur.archlinux.org/packages/gazebo/#comment-711597